### PR TITLE
MAIN - expands footer props 

### PR DESCRIPTION
**Pull request by paschalidi**
expands props 
1. `position: PropTypes.oneOf(["left", "center"])` for aligning on the left side when needed.
2.  `withBorder: PropTypes.bool` for adding/removing the top border when needed.

### CODE REVIEW

**Comment by paschalidi** [comment]
no border: 
image
left: 
image

**Comment by herman-rogers** [comment]
So we should create a footer that is just an empty component that fixes something to the bottom, then have children and styles passed into it (maybe in the future we need to right align it or center it). We should try to make this generic (unlike v1) so we can reuse it.

File: src/Footer/footer.style.js [hunk 1]
```diff
@@ -1,53 +1,11 @@
 import styled from "styled-components";
 
 const FooterStyles = styled.div`
```
Comment (herman-rogers) [inline src/Footer/footer.style.js]: Styles instead of FooterStyles

File: src/Footer/footer.style.js [hunk 2]
```diff
@@ -1,53 +1,11 @@
...
-    border-top: solid 0.5px #cdcdcd;
-    margin: 0 15%;
-  }
-
-  .link {
-    display: flex;
-    border-right: 1px solid #cdcdcd;
-    padding: 0 16px;
+  a {
+    display: block;
```
Comment (herman-rogers) [inline src/Footer/footer.style.js]: are you sure this doesn't do weird things?

File: src/VivyFooter/VivyFooter.jsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import Styles from "./vivyFooter.style";
```
Comment (herman-rogers) [inline src/VivyFooter/VivyFooter.jsx]: you could probably just call this file "footer.style.js" even though we have it already. We don't use it in a fashion where we'll get confused imo

Comment (paschalidi) [reply src/VivyFooter/VivyFooter.jsx]: `styles.js` ?

File: stories/Footer.story.jsx [hunk 4]
```diff
@@ -1,55 +1,172 @@
...
-  "default",
-  withInfo("The raw footer component")(() => (
-    <LayoutStyles>
-      <Footer>{footerChildren}</Footer>
-    </LayoutStyles>
-  ))
-);
+storiesOf("Footer", module)
+  .add(
+    "default",
```
Comment (herman-rogers) [inline stories/Footer.story.jsx]: let's not call this default, I mean, what does it even mean to be default? Is this the default that's always used (with the links inside), does it mean something else, will this always be imported, dunno. Maybe just Example default footer, Example default styles, etc.

Comment (herman-rogers) [reply stories/Footer.story.jsx]: Example /w default styles - dunno this is our documentation after all - we shouldn't worry about length but more about description.

Comment (paschalidi) [reply stories/Footer.story.jsx]: indeed. I renamed it to default styles

File: src/Footer/footer.style.js (hunk 2 above)
Comment (paschalidi) [reply src/Footer/footer.style.js]: margin wont work otherwise.
I tested it.

**Pull request merged**